.github/workflows/on-pull-request.yml

```diff
@@ -0,0 +1,39 @@
+name: Test app on a new pull request
+
+on:
+  push:
+    branches: [ master, ol5 ]
+  pull_request:
+    branches: [ master, ol5 ]
```

Comment:
Not related to this PR in particular, but can we please rename the branch to `main`?